pkgs/servers/web-apps/discourse/default.nix

```diff
@@ -166,8 +166,13 @@ let
       redis
       nodePackages.uglify-js
       nodePackages.terser
+      oxipng
     ];
 
+    # Without this set we get this error message:
+    # > Error running ember build
+    EMBER_CLI_PROD_ASSETS = 0;
```

Comment:
It looks like getting this to work may involve packaging a bunch of yarn applications that live at https://github.com/discourse/discourse/tree/main/app/assets/javascripts

Reply:
Probably related to not building ember assets: when I deploy this version to one of my servers it still says it is still on 2.9.0.beta1

Reply:
Sorry, I've been very busy with other stuff, but finally got some time to look into this. I have a draft solution here: https://github.com/NixOS/nixpkgs/pull/165481. Still need to make the update script update the yarn deps, but it's building and running fine for me. The fact that beta2 said beta1 was because they forgot to update the version number, so your build probably works as it should :)